### PR TITLE
release fixes

### PR DESCRIPTION
**Pull request by zramsay**
- small re-name fixes for the release pipeline

### CODE REVIEW

**Comment by zramsay** [comment]
todo drop choco support

File: misc/release/release.sh [hunk 1]
```diff
@@ -86,15 +46,16 @@ LATEST_TAG=$(git tag | xargs -I@ git log --format=format:"%ai @%n" -1 @ | sort |
 MONAX_RELEASE=1
 
 # NOTE: Set these up before continuing:
-export GITHUB_TOKEN=
-export AWS_ACCESS_KEY=
-export AWS_SECRET_ACCESS_KEY=
+# export GITHUB_TOKEN=
```
Comment (silasdavis) [inline misc/release/release.sh]: Do we want these commented or not here at all?

If commented can we put a note to say why you would uncomment.

**Comment by zramsay** [comment]
replaced by #1328 which needs the choco files removed

File: misc/release/release.sh (hunk 1 above)
Comment (zramsay) [inline misc/release/release.sh]: yeah it's fine. either they'd be uncommented and filled out (but not ever committed), or whoever is running the script for release will have them set as env vars locally

Comment (compleatang) [inline misc/release/release.sh]: usage throws if they aren't filled out and tells the user to fill them in.

**Pull request closed without merging**